packages/ui/src/components/button/base-button.tsx

```diff
@@ -31,14 +34,20 @@ export const BaseButton = React.forwardRef(function BaseButton(
     ...restProps,
   };
 
-  return (
+  const _className = clsx(
+    'relative inline-flex touch-none select-none flex-row items-center overflow-hidden transition duration-150 ease-in-out focus-visible:ring-2 focus-visible:ring-current focus-visible:ring-offset-1 focus-visible:ring-offset-bg disabled:cursor-not-allowed disabled:opacity-75 disabled:shadow-none',
+    ITEM_ALIGN_MAP[align],
+    className,
+  );
+
+  return href && !allProps.disabled ? (
+    <NextLink className={_className} href={href}>
```

Comment:
Add ref as well?

Reply:
I mean ` <NextLink className={_className} href={href} ref={ref}>`

Reply:
Added. But it's not perfect because of the wrong `HTMLButton Props` for `NextLink` component. We can do better but need more modifications. I'll put the sandbox here for future reference: https://codesandbox.io/s/button-and-anchor-kndluy?file=/src/App.tsx